composer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "utopia-php/fetch",
+    "description": "A simple library that provides an interface for making HTTP Requests.",
+    "type": "library",
+    "license": "MIT",
+    "require": {
+        "php": ">=8.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.5",
+        "laravel/pint": "^1.5.0",
+        "phpstan/phpstan": "^1.10"
+    },
+    "scripts": {
+        "test": "phpunit --verbose tests",
+        "check": "./vendor/bin/phpstan analyse --level max src tests",
+        "lint": "pint --test",
+        "format": "pint"
+    },
+    "autoload": {
+        "psr-4": {
+            "Utopia\\Fetch\\": "src/"
+        }
+    },
+    "authors": [
+        {
+            "name": "Sahil Saha",
+            "email": "sahilsaha7773@gmail.com"
+        }
+    ]
```

Comment:
As a team, we decided in the past that we will not add any individual's name to the `authors` section in our Utopia libraries. Let's remove it from here, please.

This is not to dishonest any of your work, we very much value all the contributions 🙏 We decided not to include any names to make it very clear that the success of those libraries isn't coupled with specific individuals, but instead, an amazing effort of the whole open source community.

Credits to specific contributors still appear in Github's contributor info, and commit history. You can also ink to the PR on your GitHub profile page.

Hope this isn't causing any trouble 🥺

Reply:
I totally understand that 😃